conftool/model/token_url.py

```diff
@@ -0,0 +1,16 @@
+from .base import BaseModel
+from .exception import ConfigException
+
+import validators
+
+class TokenUrlModel(BaseModel):
+    __inner_val__: str
+
+    def __init__(self, val: str = ""):
+        super().__init__()
+        self.__inner_val__ = val
+
+    def check(self):
+        if validators.url(self.__inner_val__) != True:
```

Comment:
No need to compare against `True` here.  Just use `if validators.url(self.__inner_val__):`.

Reply:
Same again.